q2_taxa/assets/package.json

```diff
@@ -23,8 +23,8 @@
     "webpack-dev-server": "^1.16.1"
   },
   "scripts": {
-    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline",
-    "build": "./node_modules/webpack/bin/webpack.js"
+    "dev": "webpack-dev-server --inline",
```

Comment:
Ah ha!

https://github.com/thermokarst/dotfiles/commit/f8091340a73a4b6390b2b09ae722566e97ef2564

I don't recall when or why that `npm-exec` line made it into my dotfiles, but that seems to be the culprit.